internal/js/page_data.js

```diff
@@ -20,11 +20,15 @@ addTime("domInteractive");
 addTime("domContentLoadedEventStart");
 addTime("domContentLoadedEventEnd");
 addTime("timeToContentfulPaint");
-addTime("timeToDOMContentFlushed");
 addTime("domComplete");
 addTime("loadEventStart");
 addTime("loadEventEnd");
 addTime("timeToFirstInteractive");
+pageData["domContentFlushed"];
```

Comment:
I'm assuming this is just holding the code from the custom metric because it won't actually work as-written.  Looks like something like this should work:
```JavaScript
try {
    if (window.performance.timing['timeToDOMContentFlushed']) {
        pageData["domContentFlushed"] = window.performance.timing.timeToDOMContentFlushed - window.performance.timing.fetchStart;
    }
} catch(e) {
}

Reply:
Thanks; I didn't honestly expect it to work either, and my JS is beyond rusty (sorry, Brendan!)

Reply:
Fixed.